wof/wofpy_config.py

```diff
@@ -25,11 +28,13 @@
 
 Options:
   -h --help     Show this screen.
-  -v --version     Show version.
+  -v --version  Show version.
+  --mode=test   deploy mode [default: test]
 """
 
-_ROOT = os.path.abspath(os.path.join(os.pardir, os.path.dirname(__file__)))
-_CONFIG = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
+_ROOT = os.path.abspath(os.path.dirname(wof.__file__))
+_CONFIG = os.path.join(_ROOT, 'examples', 'production_configs')
+_ODM2_TIMESERIES = os.path.join(_ROOT, 'examples', 'flask', 'odm2', 'timeseries')
```

Comment:
@ocefpaf From my understanding, `odm2/timeseries` is better because we can end up having `odm2/measurements` also in the future, assuming that the user wants to have multiple wofpy server config with `timeseries` and `measurements` etc. So can you keep it as `odm2/timeseries` rather than `odm2timeseries`?